apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -150,6 +151,47 @@ Page {
                 }
                 onClicked: pageStack.push("PermissionPage.qml")
             }
+
+            BrowserListItem {
+                //% "Save destination"
+                label: qsTrId("settings_browser-la-save_destination")
+                iconSource: "image://theme/icon-m-download"
+                value: {
+                    if (WebEngineSettings.useDownloadDir) {
+                        //% "Download dir"
+                        return qsTrId("sailfish_browser-me-download_dir")
+                    } else {
+                        //% "Ask"
+                        return qsTrId("sailfish_browser-me-ask")
+                    }
+                }
+
+                secondaryText: {
+                    if (WebEngineSettings.useDownloadDir) {
+                        //% "Downloaded files will be saved to %1"
+                        return qsTrId("sailfish_browser-me-will_be_saved_to_download").arg(WebEngineSettings.downloadDir)
+                    } else {
+                        //% "Always ask where save to"
```

Comment:
Should be "Always ask where to save" or "Always ask where to save to" (I prefer the former personally).